### PR TITLE
added default values for effects

### PR DESCRIPTION
**Pull request by MaxVoloskiy**
Added default values of effects. Refactored minimum and maximum values for each effect.

### CODE REVIEW

File: src/client/components/slider/slider.js [hunk 1]
```diff
@@ -2,23 +2,33 @@ import React, { Component } from 'react';
...
+import { connect } from 'react-redux';
 
 import './slider.css';
 
 class OneSlider extends Component {
   constructor(props, context) {
     super(props, context);
+    const { effectValues } = props;
+    const {
+      value, maxValue, minValue, step,
```
Comment (tonypntsrn) [inline src/client/components/slider/slider.js:13]: You ca destruct this directly from props

File: src/client/components/slider/slider.css [hunk 2]
```diff
@@ -35,11 +36,17 @@
   transition: 0.3s ease all;
 }
 
-.Sliders--label {
+.Slider--value {
+  color: #ffffff;
```
Comment (tonypntsrn) [inline src/client/components/slider/slider.css]: Why don’t you add this color to vars?

File: src/client/components/slider/slider.js (hunk 1 above)
Comment (MaxVoloskiy) [reply src/client/components/slider/slider.js:13]: fixed

File: src/client/components/slider/slider.css (hunk 2 above)
Comment (MaxVoloskiy) [reply src/client/components/slider/slider.css]: fixed

**Review by StanislavFedyna — APPROVED** [review]

**Review by tonypntsrn — APPROVED** [review]

**Pull request merged**